app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -93,6 +92,17 @@ class TopicPlayFragmentTest {
     ).check(matches(hasDescendant(withText(containsString("3 Chapters")))))
   }
 
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_storyProgress_isDisplayed() {
+    activityTestRule.launchActivity(null)
+    onView(
+      atPosition(
+        R.id.story_summary_recycler_view,
+        0
+      )
+    ).check(matches(hasDescendant(withText(containsString("100%")))))
+  }
+
```

Comment:
testTopicPlayFragment_loadFragmentWithTopicTestId0_clickStoryItem_opensStoryActivityWithCorrectIntent is failing

Reply:
Resolved this.